lib/Server.php

```diff
@@ -244,6 +242,8 @@ private function _handle($request) {
                     $one->params = array($one->params);
                 }
 
+                // The magic happens
+                $func   = $this->{$one->method};
```

Comment:
I'm a little concerned about moving this. This was addressed in dd6fbbb405a8d4218ea7a520daed720aef0e4489, and changing it will break that syntax.